docs/fundamentals/flow/flow-api.md

```diff
@@ -238,6 +238,34 @@ f = Flow(protocol='http', expose_graphql_endpont=True)
 For more details about the Jina GraphQL enpoint, see {ref}`here <flow-graphql>`.
 ````
 
+(flow-https)=
+### Enable HTTPS
+
+You can enable HTTPS connectivity to your Flow by configuring the underlying [uvicorn](https://www.uvicorn.org/) server.
+To do this, you need an `ssl_certificate`, an `ssl_keyfile`, and an `ssl_keyfile_password`, which you pass to `uvicorn_kwargs`
```

Comment:
You are mentioning uvicorn below but I would point out here as well that the cert/key/password variables have the excact same meaning as in uvicorn. I think thats not obvious?

Reply:
Good point, will clarify

Reply:
maybe link to a documentation page on uvicorn explaining these parameters?

Reply:
> maybe link to a documentation page on uvicorn explaining these parameters?

It is linked at the bottom in the see also box, but I will move it to the top, needs to be more visible